man/rgbgfx.1

```diff
@@ -124,12 +124,14 @@ If
 begins with a hash character
 .Ql # ,
 it is treated as an inline palette specification.
-It should contain a comma-separated list of hexadecimal colors, each beginning with a hash.
-Colors in are accepted either as
+It should contain a comma-separated list of hexadecimal colors (or the word "none"), each beginning with a hash.
+Colors are accepted either as
 .Ql #rgb
 or
 .Ql #rrggbb
 format.
+To leave a gap in the palette, write
+.Ql #none .
```

Comment:
Continued from above:
```suggestion
```
Alternatively, maybe leave this explicit explanation of `#none` here, and don't even mention it in the above part.

Reply:
Then that would be

```suggestion
To leave a gap in the palette,
.Ql #none
can be used instead of one or more colours.
```

Reply:
Would it be worth moving the thing about quoting this argument up a bit? Right now it implies that only semicolons have to be quoted, but it should really apply to everything to avoid `#` getting parsed as a line comment.

Reply:
Agreed! Though I think it'd be better part of a separate PR, to avoid "inflating" the scope of this one.